tests/integration/targets/omevv_baseline_profile/tests/create_single_cluster.yaml

```diff
@@ -0,0 +1,119 @@
+# Dell OpenManage Ansible modules
+# Copyright (C) 2024 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+# GNU General Public License v3.0+ (see COPYING or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+---
+- name: Test that we have an OME host, OME username and OME password
+  ansible.builtin.fail:
+    msg: "Please define the following variables: hostname, vcenter_username
+      and vcenter_password."
+  when:
+    hostname is undefined or vcenter_username is undefined or vcenter_password
+    is undefined
+
+- name: Set prerequisite
+  ansible.builtin.include_tasks: _helper_create.yml
+
+- block:
+    - name: Create a baseline profile with single
+        cluster (Check mode - Changes expected)
+      dellemc.openmanage.omevv_baseline_profile: &bp_single_cluster
+        state: present
+        name: OMAM_Baseline_Profile
+        repository_profile: OMAM_Firm_Repo
+        cluster: Raj-Cluster-1
```

Comment:
don't hardcode cluster

Reply:
This will be prerequisite. We need to have static name as OMAM-Cluster-1. No other choice i see. If anything, will do accordingly

Reply:
Raj, fetching cluster name dynamically should be pre-requisite.

Reply:
This will be prerequisite before running the suite